Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -3891,8 +3891,13 @@ def list_risky_users_or_host_command(client: CoreClient, command: str, args: dic
             outputs = client.risk_score_user_or_host(id_).get('reply', {})
         except DemistoException as e:
             if error_message := enrich_error_message_id_group_role(e=e, type_="id", custom_message=""):
-                raise DemistoException(error_message)
-            raise
+                user_not_found_message = f'{id_} was not found'
```

Comment:
why is this throwing an error in the first place? 404? Lets pass 404 as part of the ok codes then so this wont be an exception